### DISCUSSION TITLE
Cannot remove the creation button on the Many to One field in Directus Cloud

### DISCUSSION THREAD

**Discussion by alejandrombc** [directus/directus#14731] [Feature Requests]
Hi!, actually dont know if this is a bug or a missing feature (I consider it a bug because I have seen this behavior on other relation types on directus). 

In short, the problem is that I want to "remove" the creation button on the Many-to-One relationship, but the UI does not show any way to handle this. This just on the "Many" part of the relationship.

My goal here is to let people **select only** the values already created on the other collection (without the ability to create new ones).

<img width="809" alt="image" src="https://user-images.githubusercontent.com/9141437/181650881-c11b126b-fa7d-4b3c-92fa-91d95ff083e6.png">

Can anyone help me?, is this a missing feature?, or maybe am not seeing something obvious 😭.

Thanks in advance!

**Comment by azrikahar** [comment]
It is a feature request in #11903 🙂 Good news is there's now a PR for it in #14320.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
